core/app/components/forms/AddRelatedPubsPanel.tsx

```diff
@@ -94,10 +40,10 @@ export const AddRelatedPubsPanel = ({
 				<Button
 					type="button"
 					data-testid="add-related-pub-button"
-					onClick={handleAdd}
+					onClick={handleUpdate}
 					className="flex-1 bg-blue-500 hover:bg-blue-600"
 				>
-					Add
+					Update
```

Comment:
Note: I changed this panel's behavior so that it 'updates' rather than only 'add's. previously, the table only showed pubs that weren't already linked. Now, since we are just showing all the pubs from `getPubsWithRelatedValues`, we show all pubs, and the ones that are linked are checked. I think this behavior makes a little more sense, but I did have to add logic for removing from based on unselecting rows

Reply:
yes, i noticed this, very nice! that's indeed how i would expect it to work